src/libraries/Microsoft.Extensions.Configuration.Abstractions/ref/Microsoft.Extensions.Configuration.Abstractions.cs

```diff
@@ -4,8 +4,17 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System;
+
 namespace Microsoft.Extensions.Configuration
 {
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]
```

Comment:
We need to follow the instructions in [updating-ref-source.md](https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/docs/coding-guidelines/updating-ref-source.md#for-most-assemblies-within-libraries) to update this file.

I don't think your diff is generated from that, is it?

Reply:
It was not. I'll revert the ref change then update it per instructions. Thanks @maryamariyan.